.eslintrc

```diff
@@ -16,6 +17,13 @@
     "flowtype/require-return-type": ["error", "always", {"excludeArrowFunctions": true}],
     "require-await": "error",
     "no-return-await": "error",
-    "sort-keys": "off"
+    "sort-keys": "off",
+    "prettier/prettier": ["error", {
+      "singleQuote": true,
+      "trailingComma": "all",
+      "bracketSpacing": false,
+      "printWidth": 120,
+      "parser": "flow"
+    }]
   }
 }
```

Comment:
What do you think about adding https://github.com/prettier/eslint-config-prettier to avoid future conflicts in eslint rules?